### PR TITLE
feat(articles):added the correct details in article card

### LINKED ISSUE #46: Render info inside all articles
**Is your feature request related to a problem? Please describe.**
The articles in top news and today news are missing some information.

**Describe the solution you'd like**
Add the rest of the info to the articles.

### PR DESCRIPTION
**Pull request by JorgeLVilla**
## Changes
_List Changes Introduced by this PR_
1. Changes the articles cards in top and today news to match the articles in navbar section.
2. Adds source for news article.
3. Adds a click to see more hover.

## Purpose
This gIves all the articles the same look.

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #46

### CODE REVIEW

File: modern-news-app/src/components/TodayNews.tsx [hunk 1]
```diff
@@ -13,21 +12,33 @@ interface MainProps {
 const TodayNews: React.FC<MainProps> = ({ todayNewsData }) => {
 
     let ArticleSize = OneThirdArticle
-    let articleIndex = 0
+    // let articleIndex = 0
```
Comment (Lap343) [inline modern-news-app/src/components/TodayNews.tsx:15]: oop you got a comment

Comment (BlackCubes) [inline modern-news-app/src/components/TodayNews.tsx:15]: Do ye need dis still?

Comment (krferrell) [inline modern-news-app/src/components/TodayNews.tsx:15]: This comment needed?

**Review by Lap343 — APPROVED** [review]
looks good after the comment

**Review by BlackCubes — APPROVED** [review]
Looks gewd! Just remove the comment, if can be

**Review by krferrell — APPROVED** [review]
Solid work Jorge!

File: modern-news-app/src/components/TodayNews.tsx (hunk 1 above)
Comment (JorgeLVilla) [reply modern-news-app/src/components/TodayNews.tsx:15]: Getting rid of it, thank you!

Comment (JorgeLVilla) [reply modern-news-app/src/components/TodayNews.tsx:15]: Getting rid of it, thank you!

**Review by gonzalespaulb — APPROVED** [review]

File: modern-news-app/src/components/TodayNews.tsx (hunk 1 above)
Comment (JorgeLVilla) [reply modern-news-app/src/components/TodayNews.tsx:15]: Getting rid of it, thank you!

**Pull request merged**